x-pack/plugins/cases/server/client/factory.ts

```diff
@@ -119,8 +120,11 @@ export class CasesClientFactory {
       excludedExtensions: [SECURITY_EXTENSION_ID],
     });
 
+    const savedObjectsSerializer = savedObjectsService.createSerializer();
```

Comment:
We now use top hits aggregations that return a raw elasticsearch version of a saved object document. To interact with that document we need to convert it to a saved object. This serializer does the conversion for us so we're passing it through to the user action service.